scripts/rspec/rspec.ps1

```diff
@@ -201,11 +207,12 @@ function CreateStringResources($lang, $rules) {
         }
     }
 
+    [void]$resources.Add("HelpLinkFormat=https://rules.sonarsource.com/$($helpLanguageMap.Get_Item($lang))/RSPEC-{0}")
+    [void]$resources.Add("Language=$($roslynLanguageMap.Get_Item($lang))")
```

Comment:
probably RoslynLanguage is better, just Language could be confused with localization language...